### PR TITLE
python: multiple `options` do not merge

### PR DESCRIPTION
**Pull request by HofiOne**
Fix for #4482 

Signed-off-by: Hofi <hofione@gmail.com>

### CODE REVIEW

**Review by alltilla — APPROVED** [review]
Nice!

---
[optional]

Can you add a simple unit test for this?

**Comment by HofiOne** [comment]
> [optional]
> 
> Can you add a simple unit test for this?

Done.

File: modules/python/tests/test_python_options.c [hunk 1]
```diff
@@ -219,6 +217,30 @@ Test(python_options, test_python_options)
...
+
+  gchar *script2 = g_strdup_printf("%s%s", script, script);
+  PythonOptions *options2 = _generate_options_test_options();
+  python_options_add_options(options2, options);
+  python_options_release(options);
+
+  _run_py_options_test(options2, script2);
+
+  g_free(script2);
+  python_options_free(options2);
```
Comment (alltilla) [inline modules/python/tests/test_python_options.c:245]: Sorry if I am missing something, but I do not how this test case works.

We test an `Options` object with every option types, then we are merging it with the same values and running the test two more times on it?

I think we should test `add_options()` with options that are not already present in the `Options` object. Also I do not understand the reason behind `gchar *script2 = g_strdup_printf("%s%s", script, script);`, it should be enough to run the script once.

Comment (HofiOne) [reply modules/python/tests/test_python_options.c:245]: oh, I see, did not take into account that these are named params
will fix it

Comment (alltilla) [reply modules/python/tests/test_python_options.c:245]: Ah, I see why you did that :) Yes they are named. Thanks for changing it!

Comment (HofiOne) [reply modules/python/tests/test_python_options.c]: hope it's fine now

**Pull request merged**